.github/workflows/build-test-lint.yml

```diff
@@ -0,0 +1,29 @@
+name: Build, Test, and Lint
+on:
+  push:
+    branches: [ "master" ]
+  pull_request:
+    branches: [ "master" ]
+
+jobs:
+  build-and-test:
+    runs-on: ubuntu-latest
+    container: kjy5/lost:latest
+
+    steps:
+    - uses: actions/checkout@v3
+    - name: 🛠 Build
+      run: make -j$(($(nproc)+1))
+    - name: 📊 Unit Tests
+      run: make test -j$(($(nproc)+1))
+    - name: 📊 README Example Tests
+      run: sh ./test/scripts/readme-examples-test.sh
```

Comment:
Why is the readme examples test listed separately rather than included in the `make test` target?

Reply:
Because... I didn't think of it? I can go change that